internal/artifact/install/install.go

```diff
@@ -127,7 +127,7 @@ func (o *artifactInstallOptions) RunArtifactInstall(ctx context.Context, args []
 		}
 
 		// Extract artifact and move it to its destination directory
-		err = utils.ExtractTarGz(f, destDir)
+		_, err = utils.ExtractTarGz(f, destDir)
```

Comment:
are we planning to use the return result?

Reply:
Yes, in the upcoming PR we use it!